code/modules/reagents/reagent_containers/food/snacks/cultural/human.dm

```diff
@@ -417,3 +462,115 @@
 	reagent_data = list(/singleton/reagent/nutriment = list("blue raspberry" = 5, "white chocolate" = 3))
 	drop_sound = 'sound/items/drop/glass.ogg'
 	pickup_sound = 'sound/items/pickup/glass.ogg'
+
+
+// Eridani
+
+/obj/item/reagent_containers/food/snacks/bowl
+	name = "a bowl of item"
+	desc = "If you're seeing this, something has gone wrong D:"
+	icon = 'icons/obj/item/reagent_containers/food/cultural/human.dmi'
+	icon_state = "puffpuffbowl"
+	trash = /obj/item/trash/snack_bowl
+	var/vendingobject = /obj/item/reagent_containers/food/snacks/puffpuff
```

Comment:
DMdoc this

Reply:
not sure what that means?

Reply:
https://github.com/shiptest-ss13/Shiptest/blob/master/.github/AUTODOC_GUIDE.md

Basically, write the documentation on what this does, either as a single line (///) or multiline (/**)
You can find examples by searching the aforementioned two in the code

Reply:
Alright, i think I did it? If I understood correctly. Please check if I did what you needed me to do. Thank you!

Reply:
The DMdoc goes above the variable, not below

```suggestion
	///The item type that the bowl dispenses, an `/obj`
	var/vendingobject = /obj/item/reagent_containers/food/snacks/puffpuff
```